certbot/plugins/standalone.py

```diff
@@ -184,7 +200,7 @@ def __init__(self, *args, **kwargs):
     def add_parser_arguments(cls, add):
         add("supported-challenges",
```

Comment:
Yes. Plugin flags are prefixed with the name of the plugin [here](https://github.com/certbot/certbot/blob/master/certbot/plugins/common.py#L69).